webapp/src/Components/Replay/ReplayViewer/ThreeScene.tsx

```diff
@@ -281,7 +290,7 @@ export class ThreeScene extends React.PureComponent<Props> {
         const ballMaterial = new MeshPhongMaterial()
         const ball = new Mesh(ballGeometry, ballMaterial)
         ball.name = BALL_NAME
-        ball.add(new AxesHelper(150))
+        ball.add(new AxesHelper(200))
```

Comment:
can we set this to a constant or add a comment to say what this number means?

Reply:
Yeah, these are gonna go away once I add in player and free fly cams